3p/polyfills.js

```diff
@@ -21,3 +21,5 @@
 
 // This list should not get longer without a very good reason.
 import 'babel-core/external-helpers';
+
+require('core-js/modules/es6.array.from');
```

Comment:
babel-core/external-helpers

Which we might need to fork :(

It has all kinds of shit for stuff we don't actually use.

Probably better to take it now and switch to closure compiler.